qemu/tests/block_iscsi_with_specical_max_sectors.py

```diff
@@ -0,0 +1,106 @@
+"""Test IO on specific max_sector_kb of disk """
+
+import logging
+
+from avocado.utils import process
+from virttest import env_process
+from virttest import data_dir
+from virttest import utils_misc
+
+from virttest.iscsi import Iscsi
+from virttest.utils_misc import get_linux_drive_path
+
+
+def run(test, params, env):
+    """
+    Test IO on specific max_sector_kb of disk.
+
+    Steps:
+        1) Create lvs based on iscsi disk with specific max_sector_kb.
+        2) Boot vm with the lvs disks.
+        3) Login guest and do io on the disks.
+        4) Wait minutes then Check the VM still running.
+
+    :param test:   QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env:    Dictionary with test environment.
+    """
+
+    def _setup_lvs(dev):
+        cmd = params['cmd_set_max_sector'].format(dev.replace("/dev/", ""))
+        process.run(cmd, shell=True)
+        cmd = params['cmd_setup_vg'].format(dev)
+        process.run(cmd, shell=True)
+        for lv in lvs:
+            cmd = params['cmd_setup_lv'].format(lv)
+            process.run(cmd, shell=True)
+            cmd = params['cmd_build_img'].format(lv)
+            process.run(cmd, shell=True)
+
+    def _cleanup_lvs(dev):
+        if vm and vm.is_alive():
+            vm.destroy()
+
+        if not dev:
+            return
+
+        cmd = params['cmd_clean_lv']
+        process.run(cmd, shell=True)
+        cmd = params['cmd_clean_vg'].format(dev)
+        process.run(cmd, shell=True)
+
+    def _execute_io_in_guest():
+        all_cmd = []
+        for serial in lvs:
+            drive = get_linux_drive_path(session, serial)
+            cmd = guest_cmd.format(drive)
+            all_cmd.append(cmd)
+
+        for cmd in all_cmd:
+            logging.info("Run io in guest: %s", cmd)
+            dd_session = vm.wait_for_login(timeout=timeout)
```

Comment:
We open several sessions to run the commands, is that required? or can we use one session to run the commands?

Reply:
Yes, there are 4 dd process run on 4 disks meanwhile.